.rubocop.yml

```diff
@@ -20,6 +21,11 @@ Naming/FileName:
   Exclude:
   - "lib/ruby-lsp.rb"
 
+RubyLsp/UseLanguageServerAliases:
+  Exclude:
+    - "test/**/*.rb"
+    - "lib/ruby_lsp/server.rb"
```

Comment:
Why is server excluded?

Reply:
It is where the aliases are defined. I've changed to disable the cop in-line on that file.